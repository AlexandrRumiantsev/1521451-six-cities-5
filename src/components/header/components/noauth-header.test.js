
import React from "react";
import renderer from "react-test-renderer";

import NoauthHeader from "./noauth-header";

describe(`Render NoauthHeader`, () => {

  it(`Render NoauthHeader`, () => {
    const NoauthHeaderTest = renderer.create(
        <NoauthHeader />
    ).toJSON();

    expect(NoauthHeaderTest).toMatchSnapshot();

  });

});
