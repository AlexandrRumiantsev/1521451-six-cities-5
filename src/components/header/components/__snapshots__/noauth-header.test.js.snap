// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NoauthHeader Render NoauthHeader 1`] = `
<div
  className="header__nav-link header__nav-link--profile"
>
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  />
  <span
    className="header__user-name user__name"
  >
    <a
      href="/login"
      onClick={[Function]}
    >
       Sign In 
    </a>
  </span>
</div>
`;
